com.unity.render-pipelines.core/ShaderLibrary/TextureStack.hlsl

```diff
@@ -199,13 +177,13 @@ float4 SampleVT_##layerSamplerName(StackInfo info, int lodCalculation, int quali
         vtProperty.layerIndex[3] = layer3Index; \
         \
         vtProperty.cacheLayer[0].TextureArray = stackName##_c##layer0Index; \
-        vtProperty.cacheLayer[0].Sampler = sampler##stackName##_c##layer0Index;\
+        vtProperty.cacheLayer[0].Sampler = cacheSampler;\
```

Comment:
Is it possible to declare VT_SAMPLER in the file above, and use it directly here, instead of taking it as a parameter?

Reply:
good idea, this can indeed be a lot simpler.
Pushed an updated version